app/views/exercise_solutions/_kids_results.html.erb

```diff
@@ -3,7 +3,7 @@
   @stats = @guide.stats_for(current_user)
 %>
 
-<div class="<%= "mu-kids-callout-#{class_for_status assignment.status}" %>">
```

Comment:
I'd not rename the `status` message for and assignment, but just aliasing the `submission_status` . That way we minimize changes, keep code compact and we still deal with the idea of a assignment status - which is the submission status, right now.

Reply:
I only changed this where both assignment and discussion could use it, and i need it to be called submission_status as status means something different for a discussion.

To be fair, kids layout isn't _discussionable_ yet, but i wanted to change it now to avoid future issues.
Also we need to define a name for this concept, so i can change it in the layouts. Submittable is already taken but i think it is the only suitable one.

Reply:
As discussed, we could simply pass the submission to this view partials.